### ISSUE TITLE
Weird secondary titlebar on Windows

### ISSUE THREAD

**Issue by trha** [ChrisRM/material-theme-jetbrains#864]
### Summary

A secondary titlebar appears on Intellij 2018.2 with Material Theme installed.

![image](https://user-images.githubusercontent.com/9519031/43362455-061a6012-931d-11e8-84ec-d252d3ecee99.png)

### Expected behavior

The secondary titlebar shouldn't exist.

### Actual behavior

A secondary titlebar appears just under the real titlebar.

### Steps to reproduce (if needed)
1. Install Material Theme.

### Environment
System
```
IntelliJ IDEA 2018.2 (Ultimate Edition)
Build #IU-182.3684.101, built on July 24, 2018
JRE: 1.8.0_152-release-1248-b8 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0.17134.191
Material Theme 2.7.0
```

material_custom_theme.xml
```
<application>
  <component name="MaterialCustomThemeConfig">
    <option name="treeSelectionColor" value="0a84ff" />
    <option name="highlightColor" value="0a84ff" />
    <option name="secondBorderColor" value="2a373e" />
    <option name="tableSelectedColor" value="0a84ff" />
    <option name="contrastColor" value="1e1e1e" />
    <option name="disabledColor" value="4e4e4e" />
    <option name="secondaryBackgroundColor" value="000000" />
    <option name="buttonColor" value="2e3c43" />
    <option name="selectionForegroundColor" value="ffffff" />
    <option name="selectionBackgroundColor" value="0a84ff" />
    <option name="textColor" value="dedede" />
    <option name="foregroundColor" value="ffffff" />
    <option name="backgroundColor" value="000000" />
  </component>
</application>
```

material_theme.xml
```
<application>
  <component name="MaterialThemeConfig">
    <option name="version" value="2.7.0" />
    <option name="selectedTheme" value="DARKER" />
    <option name="highlightColor" value="80cbc4" />
    <option name="isContrastMode" value="true" />
    <option name="isMaterialDesign" value="false" />
    <option name="isBoldTabs" value="true" />
    <option name="isCustomTreeIndentEnabled" value="true" />
    <option name="accentColor" value="0a84ff" />
    <option name="useMaterialIcons" value="false" />
    <option name="useProjectViewDecorators" value="false" />
    <option name="statusBarTheme" value="false" />
    <option name="tabsHeight" value="18" />
    <option name="isCompactStatusBar" value="true" />
    <option name="isCompactTables" value="true" />
    <option name="accentScrollbars" value="false" />
    <option name="useMaterialFont" value="false" />
    <option name="compactDropdowns" value="true" />
    <option name="upperCaseButtons" value="false" />
    <option name="accentTitleBarColor" value="80cbc4" />
    <option name="userId" value="3eb0de79:16407f022d0:-7fff" />
    <option name="allowDataCollection" value="true" />
    <option name="isHighContrast" value="true" />
  </component>
</application>
```

**Comment by mallowigi** [comment]
Can you turn on "Material Components" (or change isMaterialDesign to true)?

**Comment by trha** [comment]
Enabling Material Design Components does fix the issue. But I'm not a fan of MD in general. I installed the plugin just for the darker theme.

**Comment by norbert-wieruszewski** [comment]
@mallowigi Turning on "Material Components" helped me. Thanks!

**Comment by mallowigi** [comment]
I'll try to fix this for the disabled components state

**Comment by mallowigi** [comment]
will be fixed in 2.7.2
